src/api/handlers/guild.ts

```diff
@@ -304,6 +305,52 @@ export function emojiURL(id: string, animated = false) {
   return `https://cdn.discordapp.com/emojis/${id}.${animated ? "gif" : "png"}`;
 }
 
+/**
+ * ⚠️ **If you need this, you are probably doing something wrong. Always use cache.guilds.get()?.emojis
```

Comment:
Move this at the end of the comment block, the description of the function should come first.